BasicSample/Assets/Shared/Scripts/ProjectConfiguratorWindow.cs

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.OpenXR.Remoting;
+using System;
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+using UnityEditor.XR.Management;
+using UnityEditor.XR.OpenXR.Features;
+using UnityEngine;
+using UnityEngine.XR.Management;
+using UnityEngine.XR.OpenXR;
+using UnityEngine.XR.OpenXR.Features;
+
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    enum ConfigurationSelection
```

Comment:
What is a better name for this enum?   What is the use of "None"?  Do we need to somehow detect which value it is from the current project setting and decide what is the "enum value for current project"?

If we do not want to pay the cost of "detect the current value of combobox", then we shouldn't use "combobox" as UI control.   If you intent to only "make changes",  it should be a list of buttons.   like this:

Change the Unity project settings for Mixed Reality Scenario, 
    [Native app running on HoloLens 2]
    [Win32 app running on PC VR]
    [Holographic Remoting remote UWP app]
    [Holographic Remoting remote Win32 app]

With 4 buttons, you only need to care about "actions" without keeping any states of "what is current projection setting".

Reply:
I think you have a great point here Oliver, I was struggling a little bit on how to go about this as it is not easy to find the current value of the combobox given how OnGui works in unity. Let me check if radio buttons work effectively for us

Reply:
addressed